ui/src/components/Header/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from "react-router-dom";
+import SearchBar from "./SearchBar";
+import BreadCrumbs from "./BreadCrumbs";
+
+import React from "react";
+import {Nav,HeaderBar} from './styles';
```

Comment:
You using prettier or something similar? noticing inconsistent formatting.

Reply:
for now only added .prettierrc so when i save
i need to run following but this will change lots of files so i will delay running this. but at least now i added prettier rules and formatOnSave
npx prettier --write api
npx prettier --write ui